### DISCUSSION TITLE
Using same wallet on two computers possible?

### DISCUSSION THREAD

**Discussion by nico89-exp** [DeFiCh/app#804] [Q&A] [answered]
Hi,

I have the same wallet on two computers for redundancy reason (e.g. if I need to re-sync on one computer I can use the wallet on the other). I never have the two Defi Wallet apps active at the same time, only one at a time. 

Question: is it fully safe to do this? 
Background of the question: Yesterday both versions where fully synced and seemed ok. Today one of the versions is missing transactions of rewards from the last 12 hours. The other versions has more rewards. I wonder if this happen due to the parallel usage and if this might cause some strange effects (e.g. regarding change addresses, etc.). Might it happen that both versions get out of sync among them?

Thanks!

**Comment by izzycsy** [comment, marked as answer]
Hi @nico89-exp , running the same wallet on 2 computers should be possible. If you're facing missing transactions history, it could either be you're facing this [issue](https://github.com/DeFiCh/app/issues/529), or it could be both apps are trying to sync to 100% using the same internet, or it could be as you mentioned change in address — which has to do with your user behaviour. If you're running the exact same wallet with the same addresses then it's possible.

**Reply by nico89-exp** [reply]
Hi @izzycsy, thanks for getting back to me. I wonder if the app on both computers (that are never active at the same time) create different change addresses when I do transactions. I use both computers to send DFI from time to time.

1. Is there a way to find out all addresses used in my wallet? Also including ischange: true?

2. further, the issue you mentioned was not solved so far, right?

3. Where does the app store transaction data? Is there a way to copy them from one to the other computer? 

4. Just a side question: when I use the addnode "xxxx" add command: will this node xxx be connected automatically at the next start of the app? Or do I have to add it to defi.conf?

Many questions, I know ....

**Reply by izzycsy** [reply]
1. `listaccounts {} false false true`
2. Issue not resolve yet
3. Transactions are stored on the [blockchain](https://mainnet.defichain.io/#/DFI/mainnet/home), the app is fetching it. You can export data via Transactions tab.
4. No need to touch the `defi.conf` file. I wouldn't recommend touching those hidden files for your safety, unless you're technical enough. <https://defichain-wiki.com/wiki/Addnode>
